penguin/web/services/project_payloads.py

```
@@ -0,0 +1,89 @@
+"""Shared project/task payload serializers for web and MCP surfaces."""
+
+from __future__ import annotations
+
+from typing import Any, Dict, Optional, Sequence
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)